singularity-eos/eos/get_sg_eos_rho_e.cpp

```diff
@@ -54,10 +54,18 @@ void get_sg_eos_rho_e(const char *name, int ncell, indirection_v &offsets_v,
           // eos accessor
           singularity::EOSAccessor_ eos_inx(eos_v, &pte_idxs(tid, 0));
           // reset inputs
+          // JMM: Address sanitizer likes these named. My guess is
+          // that the forwarding references are not being resolved
+          // properly.
```

Comment:
There are other comments like this elsewhere

Reply:
Resolved. Though we may wish to change the cause of this comment down the line. (The forwarding references.)

Reply:
Comment is now 
```
          // JMM: The solver constructor is (deep under the hood)
          // capturing by reference. So to avoid out-of-scope access,
          // these must be "anchored" at caller scope.
```
As you say it's more than just making the sanitizer happy. This was a real bug.